sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -387,9 +388,9 @@ class CliSuite extends SparkFunSuite with BeforeAndAfterAll with Logging {
 
   test("SPARK-26321 Should not split semicolon within quoted string literals") {
     runCliWithin(3.minute)(
-      """select 'Test1', "^;^";""" -> "Test1\t^;^",
-      """select 'Test2', "\";";""" -> "Test2\t\";",
-      """select 'Test3', "\';";""" -> "Test3\t';",
+      """select 'Test1', "^;^";""" -> "Test1|^;^",
+      """select 'Test2', "\";";""" -> "Test2| \";",
+      """select 'Test3', "\';";""" -> "Test3| ';",
```

Comment:
I think the output was matched to Hive's intentionally.

Reply:
Do you mean the extra space? the minimun col width is 3